src/Entities/BarChart.php

```diff
@@ -13,17 +13,38 @@ class BarChart extends Chart
 {
     private $x_axis;
     private $y_axis;
-    private $x_order;
```

Comment:
Removed these (and the corresponding case statements), as we really can only guarantee a chart is sorted by either X or Y (since they may conflict). Opted for just 'order', which for BarChart is the x-axis